chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+if ! [[ -r `pwd` ]]; then
+    echo "Directory is not readable."
+    exit 255
+elif [[ "$#" -lt 2 ]]; then
+    echo "At least 2 args are expected"
+    exit 1
+elif [[ "$#" -gt 2 ]]; then
+    echo "Too many args, only 2 args are expected"
+    exit 1
+elif [[ -e $1 && -f $2 ]]; then
```

Comment:
What if there was a task to accept 3 or 4 files? Then you script would stop working and you will need to write it again because of this `if/else` workflow. In most of cases, if you have repeated actions for > 1 elements, you should better use `loop` implementation instead of `if/else` approach.

Loop implementation will be much easier to read and it will give you less similar lines (DRY principle). So please reorganize it in `loop` approach and please add docs and bash `functions` for better code readability

Reply:
I'm not sure about this (loop), please take a look at the new version..

Reply:
Added more comments to clarify what I mean